src/client/editor/Editor.js

```diff
@@ -1011,6 +1013,33 @@ export default class Editor {
       }
     }
 
+    const animations = [];
+
+    clonedScene.traverse(object => {
+      const gltfModelComponent = GLTFModelComponent.getComponent(object);
+      const loopAnimationComponent = LoopAnimationComponent.getComponent(object);
+
+      if (gltfModelComponent && loopAnimationComponent) {
+        const gltfRoot = object.children.find(node => node.type === "Scene");
+        const clipName = loopAnimationComponent.props.clip;
+
+        if (gltfRoot && clipName !== null) {
+          const animation = gltfRoot.animations.find(a => a.name === clipName);
+
+          if (animation) {
+            animations.push(animation);
+          } else {
+            throw new Error(`Animation for clip "${clipName}" not found.`);
+          }
+        } else if (loopAnimationComponent) {
+          const index = object.userData._components.findIndex(
```

Comment:
this seems like it should be broken out into a function in the spirit of the other component management APIs

Reply:
We need to fix this in general. We should not have to have two different `getComponent` methods just because we clone a scene on export. I think we should move to real component classes in the future that are not stored in userData. Furthermore we should stop using three's clone method on object3D's. It doesn't work properly for skinned meshes and we can implement our own method of cloning components. I'll refactor this code after the beta launch.